GoogleMapsComponents/wwwroot/objectManager.js

```diff
@@ -160,11 +160,17 @@ window.googleMapsObjectManager = {
             });
     
             //Wait for promise
```

Comment:
Captain obvious.

Reply:
I overly comment my code when I'm doing something I've never done before. Somehow I think someone else will be like: "WTH is this for?" LOL